torchgen/api/lazy.py

```diff
@@ -31,7 +31,19 @@
     SymIntT,
 )
 
-valueT = BaseCppType("torch::lazy", "Value")
+
+_valueT = None
+
+def getValueT():
```

Comment:
@bdhirsh the idea here is that we want to clean up stuff that refers to valueT or lazy::Value in our logic and make sure it all originates from one definition which xla could customize.  I think this approach is ok, but do you have any other ideas how to pass 'valueT' from the user cmdline into where it can be accessed globally by our api and dest code?

Reply:
yeah i think this approach looks ok. Maybe it's a bit unfortunate that `BaseCppType` doesn't let you separate the "identifier" (`valueT`) from the stringified output ("lazy::Value" vs something else for XLA). But this looks fine.

Do you think XLA's version of `valueT` will be around for the long term?

Reply:
> Do you think XLA's version of valueT will be around for the long term?

It seems like it will be for a while.  There is a possibility to align it, but that requires xla side to stop using the 'XLAShape' class, which i think will be possible only after they do a full migration of their ops.